ocs_ci/utility/utils.py

```diff
@@ -358,6 +358,9 @@ def run_cmd(cmd, **kwargs):
 
     Args:
         cmd (str): command to run
+        quiet (bool, optional, kw): Whether to log the current cmd execution
```

Comment:
As this is not Args I would use some method of documentation described here with Keyword arguments, WDYT?
https://stackoverflow.com/questions/1137161/what-is-the-correct-way-to-document-a-kwargs-parameter

When you need to break line, please use on second line just one indention.

Reply:
I agree. Also, noted regarding the line break. Implementing now :)